README.md

```diff
@@ -1,20 +1,38 @@
 # DevOps Apprenticeship: Project Exercise
 
-## Getting started
+## Initial Setup
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
+A file called .env has been created with the environment variables below. This .env file is used by flask to set environment variables. This enables things like development mode (which also enables features like hot reloading when you make a file change). Populate the following variables inside the .env file with your Trello App API details/credentials:
 
-### On macOS and Linux
 ```bash
-$ poetry install
+FLASK_APP=app
+FLASK_ENV=development
+
+apiKey=# your trello api key
+apiToken=# your trello api token
+boardId=# your board id
+
+TODO_LIST_ID=# your 'todo' list id
+DOING_LIST_ID=# your 'doing' list id
+DONE_LIST_ID=# your 'done' list id
+```
+Note that .env has been added to the gitignore file so that these secrets will not be commited to git. Now, our app is ready to be run.
+
+### Running the App
+```bash
+$ poetry run python -m pytest
 ```
-You'll also need to clone a new .env file from the .env.tempalate to store local configuration options. This is a one-time operation on first setup:
 
-$ cp .env.template .env  # (first time only)
+To build a docker image (in Development mode) and start the application in Development mode , run the following:
+```bash
+$ docker build --target development --tag todo-app:dev .
+$ docker run -d -p 5000:5000 --env-file ./.env todo-app:dev 
```

Comment:
This needs a bind mount to your local code so that you can edit your python files while running the dev container (it's not a great dev container if you can't change the code!)